server/pkg/webserver/controller/gateway.go

```diff
@@ -168,3 +263,45 @@ func HandleImportAction(ctx iris.Context) base.Result {
 		Data:    data,
 	}
 }
+
+func DownloadConfigFile(ctx iris.Context) base.Result {
+	id := ctx.Params().GetString("id")
+	if id == "" {
+		return base.Response{
+			Code:    base.Error,
+			Message: "id parse failed",
+		}
+	}
+	configPath := filepath.Join(config.Cfg.Web.TasksDir, id, "config.yaml")
+	ctx.SendFile(configPath, "config.yaml")
+	return base.Response{
+		Code: base.Success,
+	}
+}
+
+func DownloadLog(ctx iris.Context) base.Result {
+	path := ctx.URLParam("pathName")
+	if path == "" {
+		return base.Response{
+			Code:    base.Error,
+			Message: "path parse failed",
+		}
+	}
+	index := strings.LastIndex(path, "/")
+	if index == -1 {
+		index = strings.LastIndex(path, "\\")
+	}
+	filename := path[index:]
```

Comment:
How about `index == -1`?

Reply:
The path may be under linux or windows env.

Reply:
I means that the value of `index` maybe `-1` here, so it will panic.